eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/AmqpResponseCode.java

```diff
@@ -25,7 +25,7 @@
 
     private final int value;
 
-    private AmqpResponseCode(final int value) {
```

Comment:
I think this constructor was deliberately private to prevent code outside the class from creating an AmqpResponseCode with an arbitrary value.

Reply:
The AmqpResponseCode is an enum class. If no access modifier is specified for the constructor of an enum type, the constructor is private. So the modifier 'private' is redundant for enum constructors.

Reply:
I didn't know that was baked into the language and I guess Sreeram didn't either. Good enough!